src/core/types.d.ts

```diff
@@ -54,6 +54,14 @@ type TwoFAConfig = {
   threshold: number
 }
 
+type TestAccounts = {
+  phone: PhoneNumber
+  code: PhoneCode
+  username: Username | undefined
+  role: string | undefined // FIXME
+  currencies // FIXME
```

Comment:
what need to be fixed?

Reply:
the type is an array of currencies, should be `any`. but because this is likely to be removed with the USD integration I didn't bother defining the type properly.